eng/pipelines/docindex.yml

```diff
@@ -63,11 +63,7 @@ jobs:
         inputs:
           pwsh: true
           filePath: eng/common/scripts/Service-Level-Readme-Automation.ps1
-          arguments: >-
```

Comment:
@danieljurek lets also be sure to remove these variables from the docindex pipelines.

Reply:
* [Done](https://github.com/Azure/azure-sdk-for-net/pull/39061)
* [Done](https://github.com/Azure/azure-sdk-for-java/pull/37029)
* [Done](https://github.com/Azure/azure-sdk-for-js/pull/27306)
* [Done](https://github.com/Azure/azure-sdk-for-python/pull/32316)

Reply:
Actually done. I removed the secrets from the variable group as other secrets in there are still needed.